.appveyor.yml

```
@@ -27,6 +26,10 @@ before_build:
 build_script:
   - cmd: dotnet build build
 
+test_script:
+  - cmd: dotnet test --no-build --logger "trx;LogFileName=testresult.xml" test/Sample.Test/Sample.Test.fsproj
```

In theory, the build script should also delegate testing properly. Can you test if this works?

```suggestion
  - cmd: dotnet test --no-build --logger "trx;LogFileName=testresult.xml" build
```

---

Seems that VSTest has changed. I probably need to update my build script/project. I can look at that later if I remember 😛.

It's not too terribly important in this project, cause I only have one test project anyways.

---

Oh wait, I've just figured it out, it's bundled in the build step. I'll push a fix in a moment.

---

There we go, I think that's what we want now